galaxykit/containerutils.py

```diff
@@ -96,4 +99,19 @@ def push_image(self, image_tag):
         if self.engine == "podman":
             run_args.append(f"--tls-verify={self.tls_verify}")
 
-        return run(run_args, capture_output=True).returncode
+        return run_command(run_args)
+
+
+def run_command(run_args):
+    logger.debug(f"Run command run_args: {' '.join(run_args)}")
+    try:
+        # python 3.6 or higher
+        result = run(run_args, capture_output=True)
+    except TypeError:
+        # older versions of python
+        result = run(run_args, stderr=PIPE, stdout=PIPE)
```

Comment:
Just use this version that's compatible with all 3.x pythons, otherwise PR is good so +1 once this is simplified